src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -112,14 +112,14 @@ private void createSymbolicLink() {
 
       // Try to delete the symbolic link in case a dangling link is present
       try {
-        Process process = Runtime.getRuntime().exec("unlink " + hdfsBinaryPath);
+        Process process = new ProcessBuilder("unlink", hdfsBinaryPath).start();
```

Comment:
Instead of method chaining, it is better java style to create a separate process builder object then call start on a separate line.